blockchain_connector/Dockerfile

```diff
@@ -112,6 +116,8 @@ COPY ./common/python /project/avalon/common/python
 COPY ./common/crypto_utils /project/avalon/common/crypto_utils
 COPY ./sdk /project/avalon/sdk
 COPY ./blockchain_connector /project/avalon/blockchain_connector
+COPY ./.git /project/avalon/.git
+COPY ./.gitignore /project/avalon/
```

Comment:
Why are we copying .git and .gitignore in container image ? Reason is not obvious. Please add a comment

Reply:
Updated

Reply:
Copying .git folder and .gitignore in container image looks bit odd. It's not good to expose specific git configurations and data in each container image. Why not create a separate dockerfile and docker-compose for lint which looks better than adding (read polluting ;-) ) each docker image with lint tools . For example refer -> https://github.com/hyperledger/sawtooth-core/blob/master/docker/compose/run-lint.yaml